web/pom.xml

```diff
@@ -39,7 +41,13 @@
 					</execution>
 				</executions>
 			</plugin>
-		</plugins>
+      <plugin>
+        <artifactId>maven-jar-plugin</artifactId>
+        <configuration>
+          <skipIfEmpty>true</skipIfEmpty>
```

Comment:
Why do we need this? Jar should never be empty because we have copied the web api in /static

Reply:
[WARNING] JAR will be empty - no content was marked for inclusion!

this will fix this. content moving to static belongs to the validate phase. 
<phase>validate</phase>